modules/log6x-cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -0,0 +1,170 @@
+<5> Specify the record field to use as the payload. The PayloadKey value must be a single field path encased in single curly brackets `{}`. Example: {.bar}.
+<6> Specify the value for the "PROCID" part of the syslog message header. The value must conform with link:https://datatracker.ietf.org/doc/html/rfc5424[The Syslog Protocol]. The value can be a combination of static and dynamic values consisting of field paths followed by `||`, and then followed by another field path or a static value. The maximum length of the final values is truncated to 48 characters. You must encase a dynamic value curly brackets and the value must be followed with a static fallback value separated with `||`. Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes. Example value: foo-{.bar||"none"}.
+<7> Set the RFC that the generated messages conform to. The value can be `RFC3164` or `RFC5424`.
+<8> Set the severity level for the message. For more information, see link:https://datatracker.ietf.org/doc/html/rfc5424#section-6.2.1[The Syslog Protocol].
+<9> Set the delivery mode for log forwarding. The value can be either `AtLeastOnce`, or `AtMostOnce`.
+<10> Specify the absolute URL with a scheme. Valid schemes are: `tcp`, `tls`, `udp` and `udps`. For example: `udps://syslog.example.com:1234`.
+<11> Specify the settings for controlling options of the transport layer security (TLS) client connections.
+<12> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+<13> Specify a name for the pipeline.
+<14> The name of your service account.
+
+. Create the CR object:
++
+[source,terminal]
+----
+$ oc create -f <filename>.yaml
+----
+
+[id="cluster-logging-collector-log-forward-examples-syslog-log-source_{context}"]
+== Adding log source information to message output
+
+You can add `namespace_name`, `pod_name`, and `container_name` elements to the `message` field of the record by adding the `enrichment` field to your `ClusterLogForwarder` custom resource (CR).
+
+[source,yaml]
+----
+  spec:
+    outputs:
+    - name: syslogout
+      syslog:
+        enrichment: KubernetesMinimal: true
+        facility: user
+        payloadKey: message
+        rfc: RFC3164
+        severity: debug
+        tag: mytag
+      type: syslog
+      url: tls://syslog-receiver.openshift-logging.svc:24224
+    pipelines:
+    - inputRefs:
+      - application
+      name: test-app
+      outputRefs:
+      - syslogout
+----
+
+[NOTE]
+====
+This configuration is compatible with both RFC3164 and RFC5424.
+====
+
+.Example syslog message output with `enrichment: None`
+[source, text]
+----
```

Comment:
How about remove some fileds from this example?

Reply:
updated, wdyt?

Reply:
No, that is my expected.
 we should highlight `namespace_name`, `pod_name`, and `container_name`  in the example when enrichment is KubernetesMinimal

Reply:
done as discussed